README.md

```diff
@@ -1 +1,43 @@
-# YouTubeClonePage
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# YouTube Clone Page - Microverse Project
+
+> This is our first project for Microverse.
+
+![screenshot](./assets/SCREENSHOT.png)
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
```

Comment:
Your live demo link is not working. Kindly replace it when you're done making all the other changes. [This link](https://rawcdn.githack.com/) might help you.